source/_base.css

```diff
@@ -7,6 +7,18 @@
   animation-iteration-count: infinite;
 }
 
+.animated.loop-1 {
+  animation-iteration-count: var(--animate-loop);
```

Comment:
I couldn’t find the `--animate-loop` variable declared anywhere.
But do we really need these loop classes?

Reply:
I will double check this one, this is for the #998 feature request which is for me make sense adding as predefined.

Reply:
If we're using utility classes, variables won't be needed at all.

My thought was something like one class that could  be reusable, changing only the variable value. Something like this:

```css
:root {
  --animate-iteration: 1;
}

.animate-iterate {
  animation-iteration-count: var(--animate-iteration);
}

.button {
  --animate-iterate: 2;
/* rest of the code */
}
```

So this would favour variables over classes. That's something that we are slowly moving to but I'm not confident if devs out there are used to this new(ish) approach. What are your thoughts?

Reply:
Indeed! clean code 101 let me implement that...

Reply:
Do you think it is a good approach? Will devs get this easily? I think Bootstrap right now is using css custom variables too, so it may have a broader usage today than I think...

Reply:
> Do you think it is a good approach? Will devs get this easily? I think Bootstrap right now is using css custom variables too, so it may have a broader usage today than I think...

Yeah, as long as the code short and simple devs might consider using it....

Reply:
Cool! So variables it is!